tasks/size_contracts.js

```diff
@@ -9,6 +9,7 @@ const {
 } = require('hardhat/builtin-tasks/task-names');
 
 const SIZE_LIMIT = 24576;
+const INIT_SIZE_LIMIT = 49152;
```

Comment:
We should update the old variable names so that they have a parallel structure with the new names.  For example, `SIZE_LIMIT` => `DEPLOYED_SIZE_LIMIT`.